cookbook/controller/error_pages.rst

```diff
@@ -132,14 +145,12 @@ is undefined. The solution is to add the following check before using this funct
 Testing Error Pages during Development
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-One of the biggest hurdles of testing how do custom error pages look in your
-application is the fact that Symfony ignores them in the development environment
-and displays the default exception pages instead.
+Symfony shows the big *exception* page instead of your customized error page
+when you're in the development environem
```

Comment:
Actually, with the paragraph below, we can just remove this one completely I guess.